rector.php

```diff
@@ -38,6 +38,7 @@
             'Doctrine\ORM\EntityManager',
             'Nette\*',
             'Symfony\Component\Routing\Loader\Configurator\RoutingConfigurator',
+            'PHPUnit\Framework\TestCase',
```

Comment:
I registered it on exclusion of `StringClassNameToClassConstantRector` for as changed to `PHPUnit\Framework\TestCase::class` on rectify

https://github.com/symplify/symplify/blob/df25fcc4d4d511d714b61abce0d02615caa912b2/packages/easy-ci/packages/Testing/Autoloading/DualTestCaseAuloader.php#L21